src/glass/src/app/core/top-bar/top-bar.component.html

```diff
@@ -7,4 +7,28 @@
   <glass-breadcrumbs></glass-breadcrumbs>
   <span fxFlex></span>
   <glass-language-button></glass-language-button>
+  <button mat-icon-button
+          matTooltip="{{ 'Help' | translate }}"
+          [matMenuTriggerFor]="helpMenu">
+    <mat-icon svgIcon="mdi:help"></mat-icon>
+  </button>
 </mat-toolbar>
+
+<mat-menu #helpMenu="matMenu">
+  <a href="https://aquarist-labs.github.io/"
+     target="_blank"
+     class="glass-text-no-decoration">
+    <button mat-menu-item>
+      <mat-icon svgIcon="mdi:web"></mat-icon>
+      <span translate>Homepage</span>
```

Comment:
Hmmm I'm wondering if `Website` would fit better here?

Reply:
Agreed. "Website" seems more appropriate.

Reply:
+1 to website. Otherwise, love it!